src/controller/python/chip/utils/DeviceProxyUtils.cpp

```diff
@@ -59,4 +59,23 @@ uint32_t pychip_DeviceProxy_ComputeRoundTripTimeout(DeviceProxy * device, uint32
         ->ComputeRoundTripTimeout(System::Clock::Milliseconds32(upperLayerProcessingTimeoutMs))
         .count();
 }
+
+/**
+ * @brief
+ *
+ * This gets the MaxPathsPerInvoke supported by remote node.
```

Comment:
While you are in here, could it be possible to add getters for more of the new fields?